owning-a-smart-contract.md

```diff
@@ -3,7 +3,7 @@
 ## Tutorial Overview
 This tutorial will walk teach you how one can access restrict a method to the deployer of the smart contract itself. This is a useful tool for defining critical business logic.
 ## Prerequisites
-- Installed Aeproject - For installation steps reffer to [this tutorial](https://hackmd.aepps.com/MwMwrATA7AhjAsBaADARgGwkfKAOApogEYAm4xAnMlBBCTOsiCUA?view)
+- Installed Aeproject - For installation steps reffer to [this tutorial](https://github.com/aeternity/tutorials/blob/master/smart-contract-deployment-in-aeproject.md)
```

Comment:
just `smart-contract-deployment-in-aeproject.md` is fine, then the pages will pick it up correctly